Tests/GRPCHTTP2CoreTests/Client/Connection/GRPCChannelTests.swift

```diff
@@ -547,6 +547,159 @@ final class GRPCChannelTests: XCTestCase {
       }
     }
   }
+
+  func testLoadBalancerChangingFromRoundRobinToPickFirst() async throws {
```

Comment:
Should we add a test to assert that if no valid load balancing config was provided we fall back to pick first?